soh/src/overlays/misc/ovl_kaleido_scope/z_kaleido_item.c

```diff
@@ -350,6 +351,40 @@ void KaleidoScope_DrawItemSelect(GlobalContext* globalCtx) {
                 KaleidoScope_SetCursorVtx(pauseCtx, index, pauseCtx->itemVtx);
 
                 if ((pauseCtx->debugState == 0) && (pauseCtx->state == 6) && (pauseCtx->unk_1E4 == 0)) {
+                    if (CVar_GetS32("gMaskSelect", 0) && cursorSlot == SLOT_TRADE_CHILD &&
+                        CHECK_BTN_ALL(input->press.button, BTN_A) && (gSaveContext.eventChkInf[8] & 0x8000)) {
```

Comment:
just a nitpick but since i missed it when looking through the first time, it might be nice to rearrange this to have the eventChkInf and cvar parts together at the beginning, and the button and cursor stuff after (or vice versa)

Reply:
Good call, done. 👍